src/utils/urls.js

```diff
@@ -0,0 +1,17 @@
+export function extractBugId(url) {
+  if (url.includes('bugzilla')) {
+    const match = /id=(\d+)/.exec(url);
+    if (match) {
+      return match[1];
+    }
+    return url;
+  }
+  if (url.includes('github')) {
+    const match = /(issues|pull)\/(\d+)/.exec(url);
+    if (match) {
+      return `#${match[2]}`;
+    }
+    return url;
```

Comment:
I don't think this is necessary.

```suggestion
```

Reply:
This is if it's a link that isn't bugzilla or github so it doesn't show the link as `undefined`, just in case.